### DISCUSSION TITLE
Query Linux sensors in "User Mode"

### DISCUSSION THREAD

**Discussion by CrownKingClown** [CrowdStrike/falconpy#1064] [Q&A] [answered]
Hi, I'm currently running a query to return the sensors that are in `reduced_functionality_mode:'yes'`. Is there any way for me to query the linux hosts that are in the user mode?

**Comment by jshcodes** [comment, marked as answer]
Hi @CrownKingClown -

Try this filter: `linux_sensor_mode:'User Mode'`

Thank you for the question! 😄

**Reply by CrownKingClown** [reply]
Thanks a lot. it works very well.
